modules/common/services/audio.nix

```diff
@@ -33,6 +33,7 @@ in
     services.pipewire = {
       enable = true;
       pulse.enable = true;
+      alsa.enable = config.ghaf.development.debug.tools.enable;
```

Comment:
Why only for debugging?

Reply:
I was asked to check why bluetooth audio is not working so I enabled it to use alsa-utils for debugging purposes. For example, when the alsa plugin is enabled you can use speaker-test and the audio will go through PipeWire instead of going directly to the audio device. It's probably not needed in the release configuration because there shouldn't be any apps that might use alsa in the audiovm. All audio should go from app vms through PipeWire. But I guess it wouldn't hurt to keep it enabled either, since it's probably just a few kilobytes.